Doc/library/unittest.rst

```diff
@@ -897,7 +897,7 @@ Test cases
    .. method:: assertIs(first, second, msg=None)
                assertIsNot(first, second, msg=None)
 
-      Test that *first* and *second* evaluate (or don't evaluate) to the
+      Test that *first* and *second* are (or are not) references to the exact
```

Comment:
I'm not sure that the word "exact" adds any value in the proposed new wording. Why not simply:

>  Test that *first* and *second* are (or are not) the same object.

?

Reply:
Agreed, fixed.